website/OpenEXRFileLayout.rst

```diff
@@ -991,177 +991,858 @@ pixels. The image has two channels: G, of type ``HALF``, and Z, of type
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+------------+-------------+
+  |00         |\\0         |             |
+  +-----------+------------+-------------+
+  |63         |c           |attribute    |
+  +-----------+------------+name:        |
+  |6f         |o           |compression  |
+  +-----------+------------+             |
+  |6d         |m           |             |
+  +-----------+------------+             |
+  |70         |p           |             |
+  +-----------+------------+             |
+  |72         |r           |             |
+  +-----------+------------+             |
+  |65         |e           |             |
+  +-----------+------------+             |
+  |73         |s           |             |
+  +-----------+------------+             |
+  |73         |s           |             |
+  +-----------+------------+             |
+  |69         |i           |             |
+  +-----------+------------+             |
+  |6f         |o           |             |
+  +-----------+------------+             |
+  |6e         |n           |             |
+  +-----------+------------+             |
+  |00         |\\0         |             |
+  +-----------+------------+-------------+
+  |63         |c           |attribute    |
+  +-----------+------------+type:        |
+  |6f         |o           |compression  |
+  +-----------+------------+             |
+  |6d         |m           |             |
+  +-----------+------------+             |
+  |70         |p           |             |
+  +-----------+------------+             |
+  |72         |r           |             |
+  +-----------+------------+             |
+  |65         |e           |             |
+  +-----------+------------+             |
+  |73         |s           |             |
+  +-----------+------------+             |
+  |73         |s           |             |
+  +-----------+------------+             |
+  |69         |i           |             |
+  +-----------+------------+             |
+  |6f         |o           |             |
+  +-----------+------------+             |
+  |6e         |n           |             |
+  +-----------+------------+             |
+  |00         |\\0         |             |
+  +-----------+------------+-------------+
+  |01         |1           |attribute    |
+  +-----------+            |size         |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+            |             |
+  |00         |            |             |
+  +-----------+------------+-------------+
+  |00         | compression: None        |
+  +-----------+------------+-------------+
+  |64         |d           |attribute    |
```

Comment:
I think this should read `None` in the middle column and "Attribute value" in the right column, right?